packages/package.njk

```diff
@@ -7,13 +7,15 @@ pagination:
   alias: pkg
 eleventyComputed:
   title: "{{ pkg.name }}"
-  description: "{{ pkg.description }}"
+  description: "{{ pkg.description | default('') }}"
 ---
```

Comment:
Q: where is this used?

Reply:
Meta and OpenGraph descriptions in the head include file.

Reply:
Ah.  So that's also basically how you push content to the head section.   E.g. scripts etc.

Reply:
> But then you must use "extends" I think.

Correct. That would in general be the preferred approach anyway. Currently it doesn't matter much, but it would clean things up a little and allow us to use blocks. Which might come in handy when we add libraries or other kinds of content.